templates/tfengine/components/devops/variables.tf

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{if not (get .admins_group "exists" false)}}
+variable "admins_group" {
+  type = object({
+    customer_id  = string
+    description  = string
+    display_name = string
+    id           = string
+    owners       = list(string)
+    managers     = list(string)
+    members      = list(string)
+  })
+  description = "Group which will be given admin access to the folder or organization."
+}
+{{- end}}
+
+variable "billing_account" {
+  type        = string
+  description = "ID of billing account to attach to this project."
+}
+
+variable "parent_id" {
+  type        = string
+  description = "ID of parent GCP resource to apply the policy. Can be one of the organization ID or folder ID according to parent_type."
+  validation {
+    condition     = can(regex("^[0-9]{8,25}$", var.parent_id))
+    error_message = "The parent_id must be valid. Should have only numeric values with a length between 8 and 25 digits."
```

Comment:
I commented above but please add a link here and run `./scripts/regen.sh`.

Reply:
Done